Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8381,6 +8397,57 @@ def domain_command():
         })
 
 
+def ip_command(ips):
+    try:
+        from urllib2 import build_opener, ProxyHandler
+        from ipwhois import IPWhois
+    except ImportError as e:
+        return_error("The Docker needs to be updated to use an IP command")
```

Comment:
Nice thinking! tho I think that once you update the image in the YML when the user updates the integration XSOAR will pull the updated image no?

Reply:
@moishce What about this?

Reply:
@Itay4 wanted to do it this way.